transfermethodui/src/main/java/com/hyperwallet/android/ui/transfermethod/view/widget/AbstractMaskedInputWidget.java

```diff
@@ -69,7 +69,7 @@ String formatToApi(@NonNull final String displayValue) {
      * @return a String formatted to the specification in the mask pattern
      */
     String formatToDisplay(@NonNull final String apiValue) {
-        if (mField != null && mField.getMask() != null) {
+        if (mField != null && mField.getMask() != null && !mField.isFieldValueMasked()) {
```

Comment:
This logic is only for update flow. Make sure you are not breaking other flows

Reply:
Unable to move this condition specific to update. Will request the QA to validate the add scenario.